_vector-search/ai-search/hybrid-search/inner-hits.md

```diff
@@ -0,0 +1,1621 @@
+---
+layout: default
+title: Inner hits with hybrid query
+parent: Hybrid search
+grand_parent: AI search
+has_children: false
+nav_order: 30
+---
+
+# Inner hits with hybrid query
+**Introduced 3.0-beta**
+{: .label .label-purple }
+
+Hybrid search users can retrieve the underlying nested inner objects or child documents information by adding an `inner_hits` clause in the search request. To know more about `inner_hits`, read [Retrieve inner hits]({{site.url}}{{site.baseurl}}/search-plugins/searching-data/inner-hits/).
+During hybrid query execution, each subquery retrieves the parent documents based on the relevant hidden inner hits. Following that, parent documents from all the subqueries are normalized and combined. Then, relevant inner_hits for each parent document are fetched from the shards.
+
+The major difference between inner hits with traditional query and hybrid query is as follows:
+1. The parent documents order in the final search response of traditional query will be determined by the inner_hits score.
+2. The parent documents order in the final search response of hybrid query will be determined by the hybrid score (i.e. normalized score). However, the parent documents are fetched from the shard based on the inner hits relevancy. 
+
+The inner hits will reflect the raw scores (i.e. scores prior to normalization) information and parent documents will reflect the hybrid score in the final search response.
```

Comment:
We should also bold this statement

Reply:
We have {: .note} added below this statement. It will pop out this statement in a highlighted way.
for example see below screenshot how line regarding pagination is highlighted.
![Screenshot 2025-04-07 at 3 27 25 PM](https://github.com/user-attachments/assets/dc1afa9e-c195-4216-b949-3694c0328857)

Reply:
yeah looks like we can't bold in the note